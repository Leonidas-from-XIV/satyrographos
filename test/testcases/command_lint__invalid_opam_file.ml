open Core
open Satyrographos_testlib

let satysfi_package_opam =
  "satysfi-package.opam", TestLib.opam_file_for_test
    ~name:"satysfi-package"
    ~synopsis:""
    ~version:"0.1"
    ()

let satyristes =
  "Satyristes", sprintf
    {|(version "0.0.2")

(library
  (name "package")
  (version "0.1")
  (sources ())
  (opam "satysfi-package.opam")
  (dependencies ()))
|}

let files =
  [ satysfi_package_opam;
    satyristes;
  ]

let () =
  TestCommand.test_lint_command files
