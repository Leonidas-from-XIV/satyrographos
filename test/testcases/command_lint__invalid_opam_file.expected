@@test_library@@/Satyristes:3:1: (module package):
@@test_library@@/satysfi-package.opam:
Warning:
  (47) Synopsis (or description first line) should start with a capital and not end with a dot

1 problem(s) found.
